tools/ui_integration/setup_ui_integration.py

```diff
@@ -0,0 +1,160 @@
+from data_repo_client import Configuration, ApiClient, ProfilesApi, DatasetsApi, SnapshotsApi, JobsApi
+import argparse
+import subprocess
+import json
+import os
+import time
+import uuid
+
+DATASETS_TO_UPLOAD = [
+    {
+        "name": "V2F_GWAS_Summary_Statistics",
+        "tables": [
+            "ancestry_specific_meta_analysis",
+            "feature_consequence",
+            "dataset_specific_analysis",
+            "frequency_analysis",
+            "trans_ethnic_meta_analysis",
+            "transcript_consequence",
+            "variant",
+        ],
+        "format": "json",
+        "stewards": ["JadeStewards-dev@dev.test.firecloud.org"],
+        "custodians": [],
+        "snapshot_creators": []
+    },
+    {
+        "name": "NonStewardDataset",
+        "tables": [],
+        "format": "json",
+        "stewards": [],
+        "custodians": [],
+        "snapshot_creators": ["JadeStewards-dev@dev.test.firecloud.org"]
+    }
+]
+
+
+class Clients:
+    def __init__(self, host):
+        config = Configuration()
+        config.host = host
+        token_output = subprocess.run(['gcloud', 'auth', 'print-access-token'], capture_output=True)
+        config.access_token = token_output.stdout.decode("UTF-8").strip()
+        self.api_client = ApiClient(configuration=config)
+
+    # Instantiate TDR snapshots API
+    def snapshots_api(self):
+        return SnapshotsApi(api_client=self.api_client)
+
+    def datasets_api(self):
+        return DatasetsApi(api_client=self.api_client)
+
+    def profiles_api(self):
+        return ProfilesApi(api_client=self.api_client)
+
+    def jobs_api(self):
+        return JobsApi(api_client=self.api_client)
```

Comment:
Is it overkill to recreate a client every call?  For instance, this will create a new client instance every time that you poll for the job status

Reply:
I went for speed of development over efficiency. I can fix that now.